lightpack.py

```diff
@@ -763,3 +824,12 @@ def __init__(self, command, response, expected):
 		self.command = command
 		self.response = response
 		self.expected = expected
+class CommandNotSupportedError(RuntimeError):
```

Comment:
Should be a blank line between classes.

Reply:
Agreed. Should we add blank lines for the other classes here too, or in another PEP8 PR?

Reply:
(Ah replying by email apparently didn't put the comment in the right place. Moving it to here...)

Ah, I didn't realize they were missing elsewhere. Yes, let's comply with PEP8 across the board in a separate commit after all this. I'll have some time later to have another look at everything.

Reply:
Fixed!